src/ui/qgisapp.ui

```diff
@@ -2222,6 +2222,14 @@ Acts on all layers.</string>
    <property name="text">
     <string>Offset Curve</string>
    </property>
+   <property name="toolTip">
+    <string>Offset Curve
+Creates an offset or an offset copy of a line or polygon ring:
```

Comment:
I was trying to be precise because the offset action operates on the ring so if you have a polygon with an interior and exterior ring the offset tool only offsets the selected ring. Same for if you have a multipart polygon. But my precision does break down because the copy aspect works on the feature level. Therefor it offsets the ring but creates a copy of the other rings and other parts. I am fine with geometry even though the tool doesn't work with points. We can have the nuance in the documentation rather than the tool tip.